pkgs/development/python-modules/jsonschema/default.nix

```diff
@@ -27,14 +27,14 @@
 
 buildPythonPackage rec {
   pname = "jsonschema";
-  version = "4.17.3";
+  version = "4.18.3";
```

Comment:
Maybe it'd be better to add a comment here explaining why not use 4.18.4 ? Or even better - a link to an upstream issue about the failing test?